releasenotes/notes/deprecate-execute-67a5c68b93722057.yaml

```diff
@@ -0,0 +1,20 @@
+---
+deprecations:
+  - |
+    Qiskit's :func:`~.execute_function.execute` function is deprecated.
+    This function served as a high-level wrapper around transpiling
+    a circuit with some transpile options and running it on a backend
+    with some run options. To do the same thing, you can explicitly
```

Comment:
Do you want to call out the sampler here too? The `BackendSampler` class is fairly straightforward mapping to what `execute()` does (although imo `execute()` or` transpile()` followed by `backend.run()` are much easier to work with).